### ISSUE TITLE
Bug Fix - Changes for VAMC field in Supporting Evidence

### ISSUE THREAD

**Issue by RakshindaAslam** [department-of-veterans-affairs/va.gov-team#58745]
## Issue Description
The scope of this ticket is to make the following changes around the VAMC field in Supporting Evidence page:
- The VAMC entry should be optional on this page (currently required)
- Updated content to clearly explain what is needed in this field
- The field should not force validation and require specific/accurate names of the facility
- Country field is optional and remove validation (currently required)
- Condition field is optional and remove validation (currently required)
- All fields are optional on this page
- User should be able to submit claim with no value or unvalidated value for these fields

## Need to know
Look at backend validation on this: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/76083e33f175fb00392e31f1f5f90654d05f1fd2/dist/21-526EZ-ALLCLAIMS-schema.json#L1016

https://www.sketch.com/s/5e464026-9acb-4f80-a4f9-634c7e972d19

This is based on our discussion with VBA stakeholders(Alejandro) on [May 18th](https://docs.google.com/document/d/1H9xgYJ-oAj3AWKtnzZxOI3xv_ayQKju03rkc36Emax8/edit#)
Remove requirement of entering VAMC name on the evidence screen
- Multiple issues are being reported from veterans with their inability to choose VAMC from options or enter a name and move to the next step or submit the form
- Even when the value is entered, it may be incorrect or incomplete, leading more work by VSR to check all records
- We understand that regardless of a veteran entering this value, VSRs look at all VAMCs in the system to assess if it’s relevant to the claim
We propose to either remove this field altogether or at least making it optional so the user does not see this issue.
 - Don’t validate the facilities name - VSR will do the research  
- Make it optional
- Clarify the language to what is required for this field

## Update 6/15/23
We found out that these newly optional fields are required in EVSS ([slack thread](https://dsva.slack.com/archives/C1VBAHWQL/p1686785454385369)) For now, we will move forward with only the content changes and removing auto suggest from the facilities name field. That should hopefully allow users to proceed. 

AC has been updated accordingly and we will file a subsequent story to be worked on next month, once the 3 fields are optional in EVSS.

---
## Tasks
- [x] Update automated tests
- [ ] ~~Backend tests~~
- [x] End to end tests

## Acceptance Criteria
- [x] Content updates to remove and edit text as noted in the sketch file
- [x] User will be able to enter a facility name that does not match any facility in the VAMC list
- [ ] ~~User will be able to enter a Country, Condition without validation~~
- [ ] ~~User will be able to submit the claim without entering any value for VAMC, Country, Condition~~
- [x] Verify no harmful downstream effects

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by RakshindaAslam** [comment]
@lydiahooper - do we need a separate ticket for content?

**Comment by lydiahooper** [comment]
@RakshindaAslam Nope, there is no IA change, and very minimal content change.

**Comment by RakshindaAslam** [comment]
Seth - Could be some backend validation on this: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/76083e33f175fb00392e31f1f5f90654d05f1fd2/dist/21-526EZ-ALLCLAIMS-schema.json#L1016

**Comment by christinec-fftc** [comment]
ICN potentially related to this issue https://dsva.slack.com/archives/C04KW0B46N5/p1686151435447459?thread_ts=1686148130.634419&cid=C04KW0B46N5

**Comment by christinec-fftc** [comment]
@lydiahooper : There is a 'Summary of evidence' page that lists out the facilities that were filled out. Is it safe to assume that we should skip this page if user does not fill out a facility name? cc @sethdarragile6 
 ![image.png](https://images.zenhubusercontent.com/641341411444ec27c3eb60a4/191a10a3-e791-4696-9839-72e162388118)

**Comment by christinec-fftc** [comment]
@lydiahooper : There is a 'Summary of evidence' page that lists out the facilities that were filled out. Is it safe to assume that we should skip this page if user does not fill out a facility name? cc @sethdarragile6 
 ![image.png](https://images.zenhubusercontent.com/641341411444ec27c3eb60a4/191a10a3-e791-4696-9839-72e162388118)

**Comment by lydiahooper** [comment]
@christinec-fftc Good question - I think we can keep this summary page, as users may still list VAMCs (even if those named are not validated)

**Comment by christinec-fftc** [comment]
Oh, I should clarify. This is for the scenario when a facility name is not specified. This is what it now looks like this when that happens (an empty bullet for each group)
![image.png](https://images.zenhubusercontent.com/641341411444ec27c3eb60a4/b73462f9-fa43-4229-8a47-fd0ea188882a)

**Comment by lydiahooper** [comment]
@christinec-fftc Right. I think the scenario in which it would be empty would be rare, because it would only happen if the user decides they don't want to provide any info about VAMCs at all. Would it be possible in this instance to simply display next to the bullet the word "unknown location"? 

We definitely don't want to skip this page as it summarizes all the actions taken in this section of the form. The language on the page before they enter VAMC info is admittedly confusing, and the design ticket I'm working on this sprint will improve that--I will make sure I look at whether we need to add this to that redesign ticket as well, so thanks for mentioning!

**Comment by christinec-fftc** [comment]
Sure thing! Yes we can put some default text.

**Comment by christinec-fftc** [comment]
@lydiahooper With @sethdarragile6's help we found out that country, treatment center name and treated disability names are all required fields for evss api. We cannot make these fields optional without the evss dev team making a corresponding change as this would cause claims submission failures. ([slack thread](https://dsva.slack.com/archives/C1VBAHWQL/p1686785454385369))

If we'd still like to proceed with making the fields optional, we'll need to make a request with the evss team (unless this is code we're allowed to change??) In the meantime, we may be able to do a short term fix for this bug by continuing with the content changes and removing autosuggest from the treatment center name. What do you think?

**Comment by lydiahooper** [comment]
@christinec-fftc I confirmed that the business agreed that the 3 fields are not required to adjudicate claims, and it looks like EVSS and Lighthouse will relax the validation rules.

**Comment by christinec-fftc** [comment]
@lydiahooper It sounds like evss changes won't be available until 7/12 in prod. do we want to wait on this story until then or proceed with short term fix (remove autosuggest) and create another story to make the fields optional next month?

**Comment by lydiahooper** [comment]
@christinec-fftc let's do what we can for now, thanks!

**Comment by christinec-fftc** [comment]
Met with @lydiahooper today for approval. Current changes look good. Created [#60578](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/60578) to make the 3 fields optional once EVSS is ready.
